spec/index.bs

```diff
@@ -1350,6 +1351,11 @@ An [=adapter=] has the following internal slots:
         A list of names of {{GPUAdapterInfo}} fields the user agent has chosen to report for this
         adapter. Initially populated with the names of any {{GPUAdapterInfo}} fields the user agent
         has chosen to report without user consent.
+
+    : <dfn>\[[behaviors]]</dfn>, of type [=ordered set=]&lt;{{GPUBehaviorName}}&gt;, readonly
+    ::
+        The [=behaviors=] this adapter supports.
```

Comment:
One big question is naming.

Is "behaviors" the right name?   It seems to fit.
I brainstormed with @dj2 about 25 names and "behaviors" was the best fit, in our opinions.

Reply:
Personally I'd prefer the name be directly relatable to some terminology in WGSL. Why not simply "WGSL software extension"? (Or renaming the WGSL concept to "behaviors" to match, but I like "software extension" better I think.)

Reply:
SGTM